march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -3,7 +3,7 @@
 #ifndef MARCH_HARDWARE_IMOTIONCUBE_H
 #define MARCH_HARDWARE_IMOTIONCUBE_H
 
-#include <map>
+#include <unordered_map>
```

Comment:
Did @JorisWeeda not notice that it mattered in what order the map was made? Or do your changes circumvent this issue?

Reply:
This bug was not noticed because when an ordered map is used all the vital components are in the first register. When an unordered map is used the bug would actually show up, because essential components would end up in the second register and would be read wrongly.